tools/eval_plots.py

```diff
@@ -669,6 +670,19 @@ def eval_plots(metrics_csv, workspace, versions = [], stats = ['CSI','FAR','TPR'
             wbd_with_metrics.to_file(Path(workspace) / 'fim_performance_polys.shp')
         else:
             print('BLE/IFC/RAS2FIM FR datasets not analyzed, no spatial data created.\nTo produce spatial data analyze a FR version')
+        
+def convert_shapes_to_csv(workspace):
```

Comment:
@BradfordBates-NOAA I'm curious why you put this .gpkg to .csv function is in eval_plots.py instead of `tools_shared_functions.py`?

Reply:
even better... to src/utils/shared_functions  (instead of the tools_shared_functions).  But.. I don't want to hold you up from merging tomorrow considering how important this is.  In lieu, would you mind just making a new card and fixing it next week if you have time.  Or someone can fix it later. Does that work?

Reply:
I think it's fine in its current script. We can move it to tools_shared_functions.py but agree with Rob that it's a minor thing. If we move it to tools_shared_functions.py then I'd need to make sure it's generic. Low priority IMO

Reply:
Definitely low priority and not suggesting to fix it now. It just seemed like an odd/unexpected place to store this function, so probably worth putting in the "clean-up later" stack.